WeakAurasOptions/BuffTrigger2.lua

```diff
@@ -33,7 +48,44 @@ local function getAuraMatchesList(name)
         end
       end
     end
-    return descText
+
+    table.sort(playerSpells)
+    table.sort(otherSpells)
+
+    if #playerSpells > 0 then
+      descText = descText .. L["Player Spells found:"]
+      for _, id in ipairs(playerSpells) do
+        addSpellToDesc(id)
+      end
+    end
+
+    if #otherSpells > 0 then
+      if descText ~= "" then
+        descText = descText .. "\n\n"
+      end
+      descText = descText .. L["Spells found:"]
+
+      for _, id in ipairs(otherSpells) do
+        addSpellToDesc(id)
+      end
+    end
+
+    local bestSuggestion
+    if #playerSpells == 1 then
+      bestSuggestion = playerSpells[1]
+    elseif #playerSpells == 0 and #otherSpells == 1 then
+      bestSuggestion = otherSpells[1]
+    end
```

Comment:
Is there any reason to make it work only when 1 spellId match instead of always returning the first of the list?

As an example, i type "power word: shield", it give me a list of ids, any of them would work